ZaneMeroff/mtv_music_trivia
# Testing 5

#### What does this PR do?
- Add tests
- Update propTypes
- Change components from class to functional

#### Where should the reviewer start?
- Open and run app
- Open and run testing suite

#### How should this be manually tested?
- Verify new tests pass and are written correctly
- Verify propTypes are all correct

#### Potential changes / Next step?
- Need to add more tests
